.eslintrc.json

```
@@ -0,0 +1,12 @@
+{
+  "env": {
+    "browser": true,
```

This should fix "chrome is not defined" error
```suggestion
    "browser": true,
    "webextensions": true,
```